plugins/out_stackdriver/gce_metadata.c

```diff
@@ -161,7 +163,7 @@ int gce_metadata_read_zone(struct flb_stackdriver *ctx)
         i++;
         j++;
     }
-    ctx->zone = flb_sds_create(zone);
+    set_resource_label("zone", zone, ctx);
```

Comment:
Wouldn't that been a waste of memory though if we had it twice? If what the metadata server is doing here is returning a string, maybe we should have these calls do exactly that instead of set things in the context itself?